src/components/HTMLEngineProvider/HTMLRenderers/MentionUserRenderer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    TNodeChildrenRenderer,
+} from 'react-native-render-html';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import Text from '../../Text';
+import Tooltip from '../../Tooltip';
+import htmlRendererPropTypes from './htmlRendererPropTypes';
+import withCurrentUserPersonalDetails from '../../withCurrentUserPersonalDetails';
+import personalDetailsPropType from '../../../pages/personalDetailsPropType';
+import * as StyleUtils from '../../../styles/StyleUtils';
+
+const propTypes = {
+    ...htmlRendererPropTypes,
+
+    /**
+     * Current user personal details
+     */
+    currentUserPersonalDetails: personalDetailsPropType.isRequired,
+};
+
+/**
+ * Navigates to user details screen based on email
+ * @param {String} email
+ * @returns {void}
+ * */
+const showUserDetails = email => Navigation.navigate(ROUTES.getDetailsRoute(email));
+
+const MentionUserRenderer = (props) => {
+    const defaultRendererProps = _.omit(props, ['TDefaultRenderer', 'style']);
+
+    // We need to remove the leading @ from data as it is not part of the login
+    const loginWhithoutLeadingAt = props.tnode.data.slice(1);
+
+    const isOurMention = loginWhithoutLeadingAt === props.currentUserPersonalDetails.login;
+
+    return (
+        <Text>
+            <Tooltip absolute text={loginWhithoutLeadingAt}>
+                <Text
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/19303
Since mention is a link (to user details), it should have used `TextLink` instead of `Text`
Not sure if this can be called a regression or a different expected behavior at the time

Reply:
Ahhhh good to know. I didn't even know about the `TextLink` component.